lib/witchcraft/apply.ex

```diff
@@ -378,8 +378,8 @@ defclass Witchcraft.Apply do
   ## Examples
 
       iex> [1, 2, 3]
-      ...> |> then([4, 5, 6])
-      ...> |> then([7, 8, 9])
+      ...> |> Witchcraft.Apply.then([4, 5, 6])
```

Comment:
Theres probably a better way to fix this so that we can keep it as `then(...)` instead of `Witchcraft.Apply.then(...)`

Reply:
When name conflict happens with Elixir's core modules like `Kernel`, I think it's better to specify full module name like this.